tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-inference-panel.html

```diff
@@ -216,7 +226,16 @@
         },
         maxClassesToDisplay: {
           type: Number,
-          value: 5,
+          value: tf_storage.getStringInitializer(
+            'maxClassesToDisplay', {defaultValue: defaultMaxClassesToDisplay}),
+          observer: 'maxClassesToDisplayChanged_',
+          notify: true,
+        },
+        samplingOdds: {
+          type: Number,
+          value: tf_storage.getStringInitializer(
```

Comment:
Nit but I think it makes more sense to use the number variant of this method: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_storage/storage.ts#L71-L75 especially since the property declared its type as `Number`.

Reply:
I initially tried to use the number variant but when trying to type in non-integers, the UI would behave strangely, between the number variant and the paper-input, leading it to be very difficult to put in numbers like 0.2. So I moved back to the string variant.